<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDetector</class>
 <widget class="QDialog" name="CreateDetector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create detector</string>
  </property>
  <property name="windowIcon">
   <iconset resource="nailab.qrc">
    <normaloff>:/Nailab/Resources/rad01.ico</normaloff>:/Nailab/Resources/rad01.ico</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="tbName">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="cbEnabled">
     <property name="text">
      <string>Enabled</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Peak locate</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Search region</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="tbSearchRegion"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Significance treshold</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="tbSignificanceTreshold"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Tolerance</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="tbTolerance"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_11">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Peak area</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Peak area region</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="tbPeakAreaRegion"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Continuum</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="tbContinuum"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Continuum function</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="cboxContinuumFunction"/>
   </item>
   <item row="10" column="0">
    <widget class="QCheckBox" name="cbUseFixedFWHM">
     <property name="text">
      <string>Use fixed FWHM</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QCheckBox" name="cbCriticalLevelTest">
     <property name="text">
      <string>95% critical level test</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QCheckBox" name="cbUseFixedTailParameter">
     <property name="text">
      <string>Use fixed tail parameter</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QCheckBox" name="cbFixSinglets">
     <property name="text">
      <string>Fit singlets</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QCheckBox" name="cbDisplayROIs">
     <property name="text">
      <string>Display ROIs</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QCheckBox" name="cbRejectZeroAreaPeaks">
     <property name="text">
      <string>Reject zero area peaks</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Max FWHMs between peaks</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLineEdit" name="tbMaxFWHMsBetweenPeaks"/>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Max FWHMs for left limit</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLineEdit" name="tbMaxFWHMsForLeftLimit"/>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Max FWHMs for right limit</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QLineEdit" name="tbMaxFWHMsForRightLimit"/>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label_13">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Background subtract</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Background subtract</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QPushButton" name="btnBackgroundSubtract">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QLineEdit" name="tbBackgroundSubtract">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QLabel" name="label_14">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Efficiency correction</string>
     </property>
    </widget>
   </item>
   <item row="20" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Efficiency calibration type</string>
     </property>
    </widget>
   </item>
   <item row="20" column="1">
    <widget class="QComboBox" name="cboxEfficiencyCalibrationType"/>
   </item>
   <item row="21" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Preset type</string>
     </property>
    </widget>
   </item>
   <item row="21" column="1">
    <widget class="QComboBox" name="cboxPresetType"/>
   </item>
   <item row="22" column="0">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>Area preset</string>
     </property>
    </widget>
   </item>
   <item row="22" column="1">
    <widget class="QLineEdit" name="tbAreaPreset"/>
   </item>
   <item row="23" column="0">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>Integral preset</string>
     </property>
    </widget>
   </item>
   <item row="23" column="1">
    <widget class="QLineEdit" name="tbIntegralPreset"/>
   </item>
   <item row="24" column="0">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>Count preset</string>
     </property>
    </widget>
   </item>
   <item row="24" column="1">
    <widget class="QLineEdit" name="tbCountPreset"/>
   </item>
   <item row="25" column="0">
    <widget class="QLabel" name="label_21">
     <property name="text">
      <string>Realtime</string>
     </property>
    </widget>
   </item>
   <item row="25" column="1">
    <widget class="QLineEdit" name="tbRealtime"/>
   </item>
   <item row="26" column="0">
    <widget class="QLabel" name="label_22">
     <property name="text">
      <string>Livetime</string>
     </property>
    </widget>
   </item>
   <item row="26" column="1">
    <widget class="QLineEdit" name="tbLivetime"/>
   </item>
   <item row="27" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="28" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="nailab.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>CreateDetector</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CreateDetector</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
